bin/cliain/src/main.rs

```diff
@@ -244,6 +247,84 @@ async fn main() -> anyhow::Result<()> {
             Ok(_) => {}
             Err(why) => error!("Unable to schedule an upgrade {:?}", why),
         },
+
+        Command::BabyLiminal(cmd) => match cmd {
+            BabyLiminal::StoreKey {
+                identifier,
+                vk_file,
+            } => {
+                if let Err(why) =
+                    store_key(cfg.get_signed_connection().await, identifier, vk_file).await
+                {
+                    error!("Unable to store key: {why:?}")
+                }
+            }
+            BabyLiminal::DeleteKey { identifier } => {
+                if let Err(why) = delete_key(cfg.get_root_connection().await, identifier).await {
+                    error!("Unable to delete key: {why:?}")
+                }
+            }
+            BabyLiminal::OverwriteKey {
+                identifier,
+                vk_file,
+            } => {
+                if let Err(why) =
+                    overwrite_key(cfg.get_root_connection().await, identifier, vk_file).await
+                {
+                    error!("Unable to overwrite key: {why:?}")
+                }
+            }
+            BabyLiminal::Verify {
+                identifier,
+                proof_file,
+                input_file,
+                system,
+            } => {
+                if let Err(why) = verify(
+                    cfg.get_signed_connection().await,
+                    identifier,
+                    proof_file,
+                    input_file,
+                    system,
+                )
+                .await
+                {
+                    error!("Unable to verify proof: {why:?}")
+                }
+            }
+        },
+
+        Command::SnarkRelation(cmd) => match *cmd {
+            SnarkRelation::GenerateSrs {
+                system,
+                num_constraints,
+                num_variables,
+                degree,
+            } => generate_srs(system, num_constraints, num_variables, degree),
+            SnarkRelation::GenerateKeysFromSrs {
+                relation,
+                system,
+                srs_file,
+            } => generate_keys_from_srs(relation, system, srs_file),
+            SnarkRelation::GenerateKeys { relation, system } => generate_keys(relation, system),
+            SnarkRelation::GenerateProof {
+                relation,
+                system,
+                proving_key_file,
+            } => generate_proof(relation, system, proving_key_file),
+            SnarkRelation::Verify {
+                verifying_key_file,
+                proof_file,
+                public_input_file,
+                system,
+            } => {
+                if verify_proof(verifying_key_file, proof_file, public_input_file, system) {
+                    println!("Proof is correct")
```

Comment:
leftover?

Reply:
After discussion with @pmikolajczyk41, we would like to leave the info about the positive outcome of the verification.